std_msgs/String imageDir
---
std_msgs/String leftFoundDir
std_msgs/String rightFoundDir
std_msgs/String leftROIDir
std_msgs/String rightROIDir
std_msgs/Int32 nleft
std_msgs/Int32 nright
